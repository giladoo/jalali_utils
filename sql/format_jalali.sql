CREATE EXTENSION jalali_utils;

SELECT format_jalali('2019-07-07 14:10:52.84937+04:30'::timestamptz, false);
SELECT format_jalali('2019-07-07 14:10:52.84937+04:30'::timestamptz);
SELECT format_jalali('2019-07-07 14:10:52.84937+04:30'::date);
SELECT format_jalali('2001-10-15'::date);
SELECT format_jalali('2021-03-20'::date);
SELECT format_jalali('2020-03-19'::date);
SELECT format_jalali('2019-03-21'::date);
DROP EXTENSION jalali_utils;
