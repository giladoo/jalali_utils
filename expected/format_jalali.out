CREATE EXTENSION jalali_utils;
SELECT format_jalali('2019-07-07 14:10:52.84937+04:30'::timestamptz, false);
 format_jalali 
---------------
 1398/04/16
(1 row)

SELECT format_jalali('2019-07-07 14:10:52.84937+04:30'::timestamptz);
    format_jalali    
---------------------
 1398/04/16 14:10:52
(1 row)

SELECT format_jalali('2019-07-07 14:10:52.84937+04:30'::date);
 format_jalali 
---------------
 1398/04/16
(1 row)

SELECT format_jalali('2001-10-15'::date);
 format_jalali 
---------------
 1380/07/23
(1 row)

SELECT format_jalali('2021-03-20'::date);
 format_jalali 
---------------
 1399/12/30
(1 row)

SELECT format_jalali('2020-03-19'::date);
 format_jalali 
---------------
 1398/12/29
(1 row)

SELECT format_jalali('2019-03-21'::date);
 format_jalali 
---------------
 1398/01/01
(1 row)

DROP EXTENSION jalali_utils;
